Libs/DICOM/Core/ctkDICOMQuery.cpp

```diff
@@ -530,22 +544,25 @@ bool ctkDICOMQuery::queryPatients()
   presentationContext = d->SCU->findPresentationContextID(UID_FINDStudyRootQueryRetrieveInformationModel, "");
   if (presentationContext == 0)
   {
-    logger.error("Failed to find acceptable presentation context");
-    emit progress(tr("Failed to find acceptable presentation context"));
+    QString error = ctkDICOMQuery::tr("Failed to find acceptable presentation context");
+    DCMTK_LOG4CPLUS_ERROR_STR(rootLogQuery, error.toStdString().c_str());
+    emit progress(error);
   }
   else
   {
-    logger.debug("Found useful presentation context");
-    emit progress(tr("Found useful presentation context"));
+    QString debug = ctkDICOMQuery::tr("Found useful presentation context");
```

Comment:
Log message should be always in English. You don't want developers to read logs full of Korean or Chinese messages. Message that is displayed to used should be translatedin general. But low-level debug messages may be better left untranslated, because translators are not developers. I would consider DICOM networking error messages to be so low-level that it is better to not translate them.